DictDataClasses.py

```diff
@@ -0,0 +1,75 @@
+import random
+import string
+import datetime
+
+class DictDataClasses:
+
+    def __init__(self):
+        self.__n_inst_ref = 14000
+        self.__n_values = 500
```

Comment:
What is this value for? Can you make it customisable through parameter passing? And then maybe have it default to 500 otherwise?

Reply:
For the number of variations in some arrays - sure i can do that